src/patternfly/components/Login/login.scss

```diff
@@ -32,44 +37,66 @@
-  --pf-c-login__main-body--c-form__group--m-action--c-button--PaddingRight: var(--pf-global--spacer--xl);
-  --pf-c-login__main-body--c-form__group--m-action--c-button--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-login__main-body--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-login__main-body--md--PaddingRight: var(--pf-global--spacer--2xl);
+  --pf-c-login__main-body--md--PaddingLeft: var(--pf-global--spacer--2xl);
+  --pf-c-login__main-body-c-form__helper-text-icon--FontSize: var(--pf-global--icon--FontSize--md);
+  --pf-c-login__main-body-c-form__helper-text-icon--MarginRight: var(--pf-global--spacer--sm);
+
+  @media (min-width: $pf-global--breakpoint--md) {
+    --pf-c-login__main-body--PaddingRight: var(--pf-c-login__main-body--md--PaddingRight);
+    --pf-c-login__main-body--PaddingLeft: var(--pf-c-login__main-body--md--PaddingLeft);
+  }
 
   // main footer
-  --pf-c-login__main-footer--PaddingTop: var(--pf-global--spacer--xl);
-  --pf-c-login__main-footer--PaddingRight: var(--pf-global--spacer--2xl);
   --pf-c-login__main-footer--PaddingBottom: var(--pf-global--spacer--3xl);
-  --pf-c-login__main-footer--PaddingLeft: var(--pf-global--spacer--2xl);
   --pf-c-login__main-footer--c-title--MarginBottom: var(--pf-global--spacer--md);
-  --pf-c-login__main-footer-link--PaddingTop: var(--pf-global--spacer--sm);
-  --pf-c-login__main-footer-link--PaddingBottom: var(--pf-global--spacer--sm);
-  --pf-c-login__main-footer-link--GridGap: var(--pf-global--spacer--sm);
-  --pf-c-login__main-footer-link--Color: var(--pf-global--Color--dark-100);
-  --pf-c-login__main-footer-link--hover--Color: var(--pf-global--Color--dark-200);
-  --pf-c-login__main-footer-grid--GridGap: var(--pf-global--gutter);
-  --pf-c-login__main-footer-grid--GridTemplateColumns: repeat(5, 1fr);
-  --pf-c-login__main-footer-grid--md--GridTemplateColumns: repeat(auto-fit, minmax(100px, 1fr));
-
-  @media (min-width: $pf-global--breakpoint--md) {
-    --pf-c-login__main-footer-grid--GridTemplateColumns: var(--pf-c-login__main-footer-grid--md--GridTemplateColumns);
-  }
+  --pf-c-login__main-footer-links--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-login__main-footer-links--PaddingRight: var(--pf-global--spacer--3xl);
+  --pf-c-login__main-footer-links--PaddingBottom: var(--pf-global--spacer--xl);
+  --pf-c-login__main-footer-links--PaddingLeft: var(--pf-global--spacer--3xl);
+  --pf-c-login__main-footer-links--MarginBottom: calc(var(--pf-c-login__main-footer-links--MarginBottom) * -1);
+  --pf-c-login__main-footer-links-item--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-login__main-footer-links-item--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-login__main-footer-links-item--MarginBottom: var(--pf-global--spacer--sm);
+  --pf-c-login__main-footer-links-item-link-svg--Fill: var(--pf-global--icon--Color--light);
+  --pf-c-login__main-footer-links-item-link-svg--Width: #{pf-size-prem(25px)};
```

Comment:
are these set to be 25px, or could you use the spacer we have for 24px?

Reply:
hmm I'm not sure. I pulled these from Andy's PR. I'm not sure we'd want to use a spacer to set the size of these though. The design does call for 24px though and we do have var for icons that size so I'd say it's ok to change it. I don't think anyone will miss 1px :)